doc/man1/cmds.pod

```diff
@@ -0,0 +1,151 @@
+=pod
+
+=head1 NAME
+
+asn1parse,
+ca,
+ciphers,
+cms,
+crl,
+crl2pkcs7,
+dgst,
+dhparam,
+dsa,
+dsaparam,
+ec,
+ecparam,
+enc,
+engine,
+errstr,
+gendsa,
+genpkey,
+genrsa,
+kdf,
+mac,
+nseq,
+ocsp,
+passwd,
+pkcs12,
+pkcs7,
+pkcs8,
+pkey,
+pkeyparam,
+pkeyutl,
+prime,
+rand,
+rehash,
+req,
+rsa,
+rsautl,
+s_client,
+s_server,
+s_time,
+sess_id,
+smime,
+speed,
+spkac,
+srp,
+storeutl,
+ts,
+verify,
+version,
+x509,
+cmds, - OpenSSL application commands
```

Comment:
`cmds` seems unnecessary

Reply:
This was one of my helpless attempts to prevent all commands to end up on the `openssl-x509` manpage. I still have no clue what I am doing wrong.

Reply:
There should be no comma after the last name...

Reply:
I'll address all your comments tomorrow...